commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/arguments/ListArgumentCommon.java

```diff
@@ -132,6 +134,7 @@ public <CommandSourceStack> List<T> parseArgument(CommandContext<CommandSourceSt
 
 		// If the argument's value is in the list of values, include it
 		List<T> list = new ArrayList<>();
+		if (skipListValidation) return list;
```

Comment:
I am unsure about this check being here. I mean, despite not checking if only valid values were given, you probably still would want access to the given list in the argument.
I'd suggest using the boolean to disable the exceptions from being thrown instead of using it to return an empty list.

Reply:
I was originally doing it this way but then discovered that the list was always empty so after some testing I decided to just have it return early instead.

Reply:
In this case, an alternate way of parsing this argument would have to be created. The list returned should contain the values provided by the user. I am not willing to say that users will have to keep track of their own list in order to validate results or something because returning an empty list is basically not acceptable.